src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
+  text-align: center;
+}
```

Comment:
**OPTIONAL**

- [ ] All style from this file is related to the App component code generated by the `create-react-app`. Feel free to remove any style that it's not going to be used by you.